### ISSUE TITLE
Checked Mode type issue with bounds

### ISSUE THREAD

**Issue by AndrewLugg** [bp74/StageXL#262]
With the commit [https://github.com/bp74/StageXL/commit/e2d9d9c6c4d7a3c59290c7bc0cf7aca776a13845](https://github.com/bp74/StageXL/commit/e2d9d9c6c4d7a3c59290c7bc0cf7aca776a13845)

We get an issue in checked mode that with this stack trace:

```
type 'Rectangle<num>' is not a subtype of type 'Rectangle<double>' of 'other' where
  Rectangle is from package:stagexl/src/geom/rectangle.dart
  num is from dart:core
  Rectangle is from dart:math
  double is from dart:core

#0      Rectangle.boundingBox (package:stagexl/src/geom/rectangle.dart:138:46)
#1      Sprite.bounds (package:stagexl/src/display/sprite.dart:104:31)
#2      DisplayObject.boundsTransformed (package:stagexl/src/display/display_object.dart:738:26)
#3      DisplayObjectContainer.bounds (package:stagexl/src/display/display_object_container.dart:360:36)
#4      Sprite.bounds (package:stagexl/src/display/sprite.dart:100:20)
#5      DisplayObject.boundsTransformed (package:stagexl/src/display/display_object.dart:738:26)
#6      DisplayObject.height (package:stagexl/src/display/display_object.dart:512:26)
```

It only occurs after an item is added as a child, to something, and then it its height or bounds are called, it then kicks this error. The old code worked fine.

**Comment by bp74** [comment]
Thanks for the error report, i will check later today!

**Comment by bp74** [comment]
Maybe it is already fixed with this commit: https://github.com/bp74/StageXL/commit/8aee81ceb449da77f4af9a6591b00991e75be31a

**Comment by bp74** [comment]
I have released version 1.1.0 which has a fix that should solve this problem. Please give it a try.
